app/views/ShareView/index.js

```diff
@@ -112,8 +114,10 @@ export default class ShareView extends React.Component {
 
 	sendTextMessage = async() => {
 		const { value, rid } = this.state;
+		const { token } = this.props;
 		if (value !== '' && rid !== '') {
 			try {
+				await RocketChat.login({ resume: token });
```

Comment:
Shouldn't you do the same for `sendMediaMessage`?
Why not login in `shareExtensionInit`?